devtools/test_dashboard/buttons.js

```diff
@@ -11,7 +13,7 @@ anchor.style.width = '1000px';
 
 function plotButtons(plots, figDir) {
 
-    Object.keys(plots).forEach( function (plotname) {
+    Object.keys(plots).forEach(function (plotname) {
```

Comment:
I'm ok adding 1 commit fixing all `function ()` instances? That should take me ~ 20 mins.